sdk/tests/conformance/textures/misc/video-rotation.html

```diff
@@ -141,9 +138,14 @@
                 "video-rotation-270",
             ];
 
-            for (let fn of filenames) {
-                fn += (prefer_vp9 ? ".vp9" : "" ) + ".mp4";
-                await testVideoElement(fn);
+            if (supports_h264) {
+                for (let fn of filenames)
+                    await testVideoElement(fn + ".mp4");
+            }
+
+            if (supports_vp9) {
+                for (let fn of filenames)
+                    await testVideoElement(fn + ".vp9.mp4");
```

Comment:
.vp9?

Reply:
No, this looks right? Sorry if I'm blind.

Reply:
Ah ok, sorry, I didn't look carefully and didn't realize these were still in an .mp4 container.